skynet/utils.py

```diff
@@ -4,13 +4,15 @@
 from fastapi import APIRouter, Depends
 
 from skynet.auth.bearer import JWTBearer
-from skynet.env import bypass_auth
+from skynet.env import bypass_auth, bypass_openai_api_auth
 from skynet.logs import get_logger, uvicorn_log_config
 from skynet.modules.monitoring import FORCED_EXIT_COUNTER
 
 log = get_logger(__name__)
 
 dependencies = [] if bypass_auth else [Depends(JWTBearer())]
+openai_api_dependencies = [] if bypass_openai_api_auth else [Depends(JWTBearer())]
```

Comment:
Isn't this redundant now? We should be able to use the same dependencies now. What'd be missing is the trick we discussed with having a UUID to skip auth.